### PR TITLE
Update 06-stewken01.md

### PR DESCRIPTION
**Pull request by stewken01**

### CODE REVIEW

**Review by Chocrates — DISMISSED** [review]
Looks great!  Can you say something about yourself?

File: _slides/06-stewken01.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Adding a caption for Kenton
```
Comment (mmahmud-nychdc) [inline _slides/06-stewken01.md:7]: Excellent work.

**Review by mmahmud-nychdc — COMMENTED** [review]
Excellent.

**Comment by stewken01** [comment]
Commenting here by kenton

**Comment by stewken01** [comment]
Looks good!

**Review by Chocrates — APPROVED** [review]

**Pull request merged**